app/src/internal/java/com/duckduckgo/app/browser/webview/WebViewDevSettingsActivity.kt

```diff
@@ -0,0 +1,127 @@
+import androidx.lifecycle.lifecycleScope
+import com.duckduckgo.anvil.annotations.ContributeToActivityStarter
+import com.duckduckgo.anvil.annotations.InjectWith
+import com.duckduckgo.app.browser.R
+import com.duckduckgo.app.browser.databinding.ActivityWebViewDevSettingsBinding
+import com.duckduckgo.app.browser.webview.WebViewDevSettingsViewModel.ViewState
+import com.duckduckgo.app.clipboard.ClipboardInteractor
+import com.duckduckgo.common.ui.DuckDuckGoActivity
+import com.duckduckgo.common.ui.viewbinding.viewBinding
+import com.duckduckgo.common.utils.playstore.PlayStoreAndroidUtils.Companion.PLAY_STORE_PACKAGE
+import com.duckduckgo.di.scopes.ActivityScope
+import com.duckduckgo.navigation.api.GlobalActivityStarter
+import javax.inject.Inject
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+
+@InjectWith(ActivityScope::class)
+@ContributeToActivityStarter(WebViewDevSettingsScreen::class)
+class WebViewDevSettingsActivity : DuckDuckGoActivity() {
+
+    @Inject
+    lateinit var viewModel: WebViewDevSettingsViewModel
+
+    @Inject
+    lateinit var clipboardManager: ClipboardInteractor
+
+    private val binding: ActivityWebViewDevSettingsBinding by viewBinding()
+
+    private lateinit var webViewDebugLauncher: ActivityResultLauncher<Intent>
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+        setupToolbar(binding.includeToolbar.toolbar)
+        registerActivityResultLauncher()
+        configureListeners()
+        observeViewModel()
+    }
+
+    private fun observeViewModel() {
+        viewModel.viewState()
+            .flowWithLifecycle(lifecycle, Lifecycle.State.CREATED)
+            .onEach { viewState ->
+                updateViews(viewState)
+            }.launchIn(lifecycleScope)
+    }
+
+    private fun updateViews(viewState: ViewState) {
+        binding.webViewPackage.setSecondaryText(viewState.webViewPackage)
+        binding.webViewVersion.setSecondaryText(viewState.webViewVersion)
+    }
+
+    override fun onStart() {
+        super.onStart()
+        viewModel.start()
+    }
+
+    private fun configureListeners() {
+        binding.webViewDevTools.setOnClickListener {
+            launchExternalIntentForAction(Intent(WEBVIEW_DEV_TOOLS_INTENT_ACTION), getString(R.string.webview_dev_ui_not_available))
+        }
+        binding.webViewVersion.setClickListener {
+            clipboardManager.copyToClipboard(viewModel.viewState().value.webViewVersion, false)
+        }
+        binding.webViewPackage.setClickListener {
+            clipboardManager.copyToClipboard(viewModel.viewState().value.webViewPackage, false)
+        }
+        binding.webViewPlayStore.setClickListener {
+            val intent = Intent(Intent.ACTION_VIEW).also {
+                it.setData(PLAYSTORE_WEBVIEW_INTENT_URI.toUri())
+                it.setPackage(PLAY_STORE_PACKAGE)
```

Comment:
@0nko FYI, just tweaked this after you approved. should remove some friction when launching the play store (e.g., won't offer to open it in Galaxy App Store)